### DISCUSSION TITLE
How can I have MSVC throw its errors?

### DISCUSSION THREAD

**Discussion by dreamer-coding** [mesonbuild/meson#13255] [Q&A]
Not that I'm trying to make things break but rather I am trying to catch issues that would normally be missed if warnings and error where turned off like the missing threads library and unused params as an example: how can get MSVC or anything else in that case to throw their error messages at me?

Heres the current pipeline:

https://github.com/fossil-lib/fscl-xtest-c/blob/main/.github/workflows/meson_ci.yml

**Comment by eli-schwartz** [comment]
That's not a pipeline. It tells me nothing about what you're looking at.

**Comment by eli-schwartz** [comment]
> if warnings and error where turned off like the missing threads library and unused params as an example

I am not really sure what you mean there. A missing threads library? If something your project needs doesn't exist, it will fail to compile. If meson didn't find it, meson will fail to configure unless you tell meson that it is optional (`required: false`).

GCC has -Werror=unused-parameter. I don't know what diagnostic warnings MSVC knows about -- that's probably a question better asked on Microsoft's compiler support forums.
